pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -333,24 +335,21 @@ func (nim *nodeInfoManager) updateCSINodeInfo(
 	retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 		nodeInfo, err := csiKubeClient.CsiV1alpha1().CSINodeInfos().Get(string(nim.nodeName), metav1.GetOptions{})
```

Comment:
You might want to use an Informer here. Will anything other than the node ever modify the node info object?

Reply:
nothing other than the node will modify the node info object. 
https://github.com/kubernetes/kubernetes/issues/70773